recipe/build.sh

```diff
@@ -18,16 +18,18 @@ export CUDAARCHS="50-real;60-real;70-real;80-real"
 if [[ "$cuda_compiler_version" == "11.2" ]]; then
   export CUDA_ARCH_LIST="${CUDA_ARCH_LIST},sm_35,sm_86"
   export CUDAARCHS="${CUDAARCHS};35-real;86"
-fi
 
-if [[ "$cuda_compiler_version" == "11.8" ]]; then
+elif [[ "$cuda_compiler_version" == "11.8" ]]; then
   export CUDA_ARCH_LIST="${CUDA_ARCH_LIST},sm_35,sm_89,sm_90"
   export CUDAARCHS="${CUDAARCHS};35-real;89-real;90"
-fi
 
-if [[ "$cuda_compiler_version" == "12.0" ]]; then
+elif [[ "$cuda_compiler_version" == "12.0" || "$cuda_compiler_version" == "12.6" ]]; then
```

Comment:
You're missing cuda 12.4 (for ppc); probably easier to just do:
```suggestion
elif [[ "$cuda_compiler_version" == 12.*  ]]; then
```